nextcloud_update.sh

```diff
@@ -1121,6 +1121,18 @@ then
     print_text_in_color "$ICyan" "Nextcloud crontab updated to run every 5 minutes."
 fi
 
+# Replace iNotify checker with a script instead
+if crontab -u www-data -l | grep -q "files_external:notify -v"
```

Comment:
Why not get the ids from the crontab? That would be a much safer way imho...

Reply:
The crontab might not be true (not used, leftover, etc), but the DB is. 

Also, will several mounts generate several crontab-rows?

Reply:
> The crontab might not be true (not used, leftover, etc), but the DB is.

However the user might not have axtivated the feature for all mounts and does not want to do so.

> Also, will several mounts generate several crontab-rows?

Yes. Each mount for which the feature was activated will have its own row.

Reply:
Hmm... It makes it a bit more complicated. 

Maybe the best option here is to just ditch the replacement to the new way of doing it.